index.html

```diff
@@ -3,6 +3,8 @@
 <head>
     <meta charset="utf-8">
     <title>Terminal 7</title>
+    <!-- Cache control meta tags -->
+    <meta http-equiv="Cache-Control" content="public, max-age=86400" />
```

Comment:
I think we've decided not to include this

Reply:
Yep , I will delete it